### ISSUE TITLE
Higher test set accuracy?

### ISSUE THREAD

**Issue by cagbal** [huguyuehuhu/HCN-pytorch#6]
Thank you for this nice repo and congrats. for your nice results. I have some questions about your repo.

~~As stated in #2, the test set accuracy is significantly higher than training accuracy during nearly first 100 epochs. Is it because of dropouts or any other reason?~~

~~Also, my second question is: are there any difference between eval accuracy during training mode and accuracy stated in --mode test? They are the same right?~~

My other question is: what is the difference between HCN[1] and HCN. Aren't they the same network with same configuration? 

HCN[1] | 32 | 256 | 86.5 | 91.1
-- | -- | -- | -- | --
HCN | 32 | 256 | 84.2 | 89.2

Edit: I understand the first two questions, so I disabled them.

**Comment by huguyuehuhu** [comment]
Sorry for the late reply. HCN[1] represents the results reported in the original paper, and HCN is our results. Because I just replicate it as a baseline for our work, we have not struggled to tune the parameters to meet the results reported in the original paper.

**Comment by cagbal** [comment]
Thanks for the reply. I thought you are one of the author.

13 Ara 2018 Per 16:51 tarihinde huguyuehuhu <notifications@github.com> şunu
yazdı:
